uint16 lift_cmd 
uint16 ir_cmd
uint16 ir_dist_left
uint16 ir_dist_right 
float32 angle
bool mount_status
bool alignment
bool busy_status